packages/monosize/src/commands/measure.mts

```diff
@@ -12,13 +12,21 @@ import { readConfig } from '../utils/readConfig.mjs';
 import type { CliOptions } from '../index.mjs';
 import type { BuildResult } from '../types.mjs';
 
-export type MeasureOptions = CliOptions & { debug: boolean };
+export type MeasureOptions = CliOptions & {
+  debug: boolean;
+  'artifacts-location': string;
+};
 
 async function measure(options: MeasureOptions) {
-  const { debug = false, quiet } = options;
+  const { debug = false, quiet, 'artifacts-location': artifactsLocation } = options;
 
   const startTime = process.hrtime();
-  const artifactsDir = path.resolve(process.cwd(), 'dist', 'bundle-size');
+  const artifactsDir = path.resolve(process.cwd(), artifactsLocation);
+
+  // thrown error if cwd is set as artifactsLocation is set to '.' since next step is to rm everything
+  if (artifactsDir === process.cwd()) {
+    throw new Error("'--artifactsLocation' cannot be the same as current working directory");
```

Comment:
Since the option was renamed to `--artifacts-location` maybe we it's better to update the error message too:
```suggestion
    throw new Error("'--artifacts-location' cannot be the same as current working directory");
```